src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -720,7 +720,7 @@ bool emberAfUnitTestingClusterTestCallback(app::CommandHandler * commandObj, con
                                            const Clusters::UnitTesting::Commands::Test::DecodableType & commandData)
 {
     // Setup the test variables
-    emAfLoadAttributeDefaults(commandPath.mEndpointId, true, MakeOptional(commandPath.mClusterId));
+    emberAfInitializeAttributes(commandPath.mEndpointId);
```

Comment:
Note: this is NOT the same as it was before. However the UnitTesting::Test command is:

```
  /** Simple command without any parameters and without a specific response */
  command Test(): DefaultSuccess = 0;
```

This seems to have been part of #13673. I think this may still be ok because we actuall re-load everything and the test is for UnitTestingCluster.

@cecille

Reply:
Seems reasonable to me.